gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_0/05_add_environment_id.yml

```diff
@@ -8,18 +8,27 @@ databaseChangeLog:
             columns:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.